### PR TITLE
Upgrades for 7.8

### PR DESCRIPTION
**Pull request by cunningt**

### CODE REVIEW

File: src/main/resources/springboot2.yaml [hunk 1]
```diff
@@ -233,10 +238,68 @@ builds:
...
+  pigYamlMetadata: Fuse dev build ${build.url} Fuse ${yaml.name} yaml config ${project.version}
+  buildScript: >
+    export M2_HOME=/usr/share/maven
+
+    mvn -e -V -B -Dmaven.test.skip.exec=true -DfailIfNoTests=false -Dtest=false -DskipTests=true -pl '!camel-rest-swagger' clean deploy
+  customPmeParameters:
+    - ${wildfly-camel-examples.pme.options}
+  dependencies:
+  - redhat-fuse-${version.fuse.prefix}
+  - wildfly-camel-5.4.0-${version.fuse.prefix}
```
Comment (orpiske) [inline src/main/resources/springboot2.yaml]: I think this should be wildfly-camel-5.6.0

File: src/main/resources/springboot2.yaml [hunk 2]
```diff
@@ -233,10 +238,68 @@ builds:
...
+  scmRevision: fuse-eap-${version.fuse.eap}
+  pigYamlMetadata: Fuse dev build ${build.url} Fuse ${yaml.name} yaml config ${project.version}
+  buildScript: >
+    export M2_HOME=/usr/share/maven
+
+    mvn -e -V -B -Dmaven.test.skip.exec=true -DfailIfNoTests=false -Dtest=false clean deploy
+  customPmeParameters:
+    - ${fuse-eap.pme.options}
+  dependencies:
+  - wildfly-camel-5.4.0-${version.fuse.prefix}
```
Comment (orpiske) [inline src/main/resources/springboot2.yaml]: I think this should be wildfly-camel-5.6.0

File: src/main/resources/springboot2.yaml [hunk 3]
```diff
@@ -233,10 +238,68 @@ builds:
...
+  - wildfly-camel-examples-5.4.0-${version.fuse.prefix}
```
Comment (orpiske) [inline src/main/resources/springboot2.yaml]: I think this should be wildfly-camel-examples-5.6.0

File: src/main/resources/springboot2.yaml [hunk 4]
```diff
@@ -233,10 +238,68 @@ builds:
...
+    #buildCommand: ${karaf-quickstarts.buildCommand}
+  project: jboss-fuse/karaf-quickstarts
+  scmUrl: git+ssh://code.engineering.redhat.com/jboss-fuse/karaf-quickstarts.git
+  externalScmUrl: ssh://git@github.com/jboss-fuse/karaf-quickstarts.git
+  scmRevision: karaf-quickstarts-${version.karaf-quickstarts}
+  pigYamlMetadata: Fuse dev build ${build.url} Fuse ${yaml.name} yaml config ${project.version}
+  customPmeParameters:
+    - ${karaf-quickstarts.pme.options}
+  dependencies:
+  - cxf-3.2.7-${version.fuse.prefix}
```
Comment (orpiske) [inline src/main/resources/springboot2.yaml]: Is this still the case? Just asking because I thought we were going to use sb2 components ...

**Review by oscerd — APPROVED** [review]

**Review by orpiske — APPROVED** [review]
Added some comments

File: src/main/resources/springboot2.yaml (hunk 4 above)
Comment (cunningt) [reply src/main/resources/springboot2.yaml]: Fixed

File: src/main/resources/springboot2.yaml (hunk 2 above)
Comment (cunningt) [reply src/main/resources/springboot2.yaml]: Fixed

**Pull request merged**